cristin-intermediate-storage/src/main/java/no/unit/nva/cristin/intermediate/storage/CristinUnitsUtil.java

```diff
@@ -0,0 +1,118 @@
+package no.unit.nva.cristin.intermediate.storage;
+
+import static io.vavr.control.Try.of;
+import static io.vavr.control.Try.ofSupplier;
+import static nva.commons.core.attempt.Try.attempt;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import io.github.resilience4j.core.IntervalFunction;
+import io.github.resilience4j.retry.Retry;
+import io.github.resilience4j.retry.RetryConfig;
+import io.github.resilience4j.retry.RetryRegistry;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.net.http.HttpResponse.BodyHandlers;
+import java.nio.charset.StandardCharsets;
+import java.util.function.Supplier;
+import no.unit.nva.commons.json.JsonUtils;
+import nva.commons.core.Environment;
+import nva.commons.core.useragent.UserAgent;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Utility class for retrieving all Norwegian cristin units where you want to do repeated lookups and keep a large
+ * cache like migrations and other bulk operations.
+ */
+public class CristinUnitsUtil {
+
+    private static final Logger logger = LoggerFactory.getLogger(CristinUnitsUtil.class);
+    private static final String API_ARGUMENTS = "/units?per_page=1000&country=NO&page=";
```

Comment:
add `?lang=nb,nn,en` for label in all supported languages. See other comment

Reply:
btw, do you know why there is problems with the encoding?
![image](https://github.com/BIBSYSDEV/nva-cristin-service/assets/554713/33a5060b-5e48-455a-9cb1-a39d24267136)

Reply:
Not sure.

Reply:
ah I retried with postman and its working fine there. seems like there is a bad default when https://api.cristin-test.uio.no/v2/units?lang=nb,nn,en&per_page=1000&page=1 is requested without headers